lib/client.ml

```diff
@@ -1,40 +1,49 @@
 open Lwt_unix
-open Util
 
-type 'a t =
-  { address: string
-  ; port: int
-  ; socket: file_descr
-  ; state: 'a ref
-  ; recv_mutex: Lwt_mutex.t
-  ; state_mutex: Lwt_mutex.t
-  }
+type 'a t = {
+  address : string;
+  port : int;
+  socket : file_descr;
+  state : 'a ref;
+  recv_mutex : Lwt_mutex.t;
+  state_mutex : Lwt_mutex.t;
+}
 
-let peer_from (client: 'a t) =
-  Peer.{ address = client.address; port = client.port}
+let peer_from (client : 'a t) =
+  Peer.
+    {
+      address = client.address;
+      port = client.port;
+      known_peers = [];
+      state = Alive;
+    }
 
 let send_to client payload peer =
   let len = Bytes.length payload in
   let addr = Peer.to_sockaddr peer in
   let%lwt _ = sendto !client.socket payload 0 len [] addr in
   Lwt.return ()
 
+let naive_broadcast client payload peers =
+  let _ = List.map (send_to client payload) peers in
+  Lwt.return ()
+
 let recv_next client =
   let open Lwt_unix in
-  (* Peek at the first 8 bytes of the incoming datagram
-  to read the Bin_prot size header. *)
-  let size_buffer = Bytes.create 8 in
+  let open Util.Encoding in
+  let buffer_size = Bytes.create size_header_length in
```

Comment:
Let's call it size_buffer, otherwise it seems like this binding is a number rather than a buffer.

Reply:
Or, to be consistent with msg_buff, we can call it size_buff

Reply:
If you want to change other truncated names to match your preference, I'm fine with that :)

Reply:
I will do it later, this is not important right now.